Modules/Loadable/Segmentations/Widgets/qMRMLSegmentEditorWidget.cxx

```diff
@@ -2757,12 +2767,24 @@ void qMRMLSegmentEditorWidget::processEvents(vtkObject* caller,
   vtkMRMLAbstractViewNode* callerViewNode = vtkMRMLAbstractViewNode::SafeDownCast(caller);
   if (callerInteractor)
     {
-    bool abortEvent = activeEffect->processInteractionEvents(callerInteractor, eid, viewWidget);
-    if (abortEvent)
+    if (eid == vtkCommand::StartPanEvent || eid == vtkCommand::StartPinchEvent || eid == vtkCommand::StartRotateEvent)
```

Comment:
Is it possible that two gestures are active at once? If yes then a single Boolean flag may not be enough to robustly determine gesture state.

Reply:
Changed the gesture flag to an integer count of gestures in progress.